### ISSUE TITLE
Difference between Parametric Pooling and Depthwise (Separable) Conv?

### ISSUE THREAD

**Issue by omiita** [megvii-model/FunnelAct#7]
Thank you for your splendid work. I really enjoyed your paper.
But, I'm little bit confused about some parts.
My questinons are below.

1. **What are you referring to by the term "Depthwise Separable Convolution"?**
Although you are using term "Depthwise **Separable** Convolution" repeatedly in your paper,
you are using "**Depthwise Convolution**" in your implementation, not Depthwise "**Separable**" Convolution.
From my understanding, "Depthwise Separable Convolution" means "Depthwise Conv. + Pointwise Conv."([Xception paper](https://arxiv.org/abs/1610.02357)) and, in your case, you seem to use only "Depthwise Conv." Am I missing something here?
> ![depthwise separable conv(§1.2)](https://imgur.com/XC6IlFN.png)
> Image from §1.2 in your paper

2. **What is the difference between ParamPool(·) and DW(·) in table 5?**
I thought ParamPool is the same as Depthwise Conv. but you are using ParamPool and DW differently in your paper. So, what is the difference between them? 
> ![Table5](https://imgur.com/K6IbPuw.png)
> Image from Table 5 in your paper

Thanks in advance!
Sincerely.

**Comment by nmaac** [comment]
Hi @omiita , we are pleased to make it more clear here:
1. we only use dw-conv without pointwise conv.
2. ParamPool=dwconv? Yes we've stated in the paper, the form extends PReLU to 2D visual activation, outperforms AvgPool as a parametric AvgPool, and is easy to implemented using dwconv operator.

**Comment by omiita** [comment]
@nmaac 
Thank you for your quick reply!
I understand.

**Comment by omiita** [comment]
Hi again, @nmaac. Sorry for reopening this issue. 
I have another question about Batch Normalization after Depthwise Convolution.

* **Why did you use Batch Normalization instead of Layer/Group Norm in your implementation?**
![table 6](https://imgur.com/oiXAGkX.png)
In ablation study, you showed LN/GN outperforms BN in ShuffleNetV2 but you didn't discuss or even mention it much.
I'm pleased to hear some explanation **why you used BatchNorm instead of other superior methods thorough your experiments(and in your implementation)**.

Thank you in advance!

**Comment by nmaac** [comment]
> Hi again, @nmaac. Sorry for reopening this issue.
> I have another question about Batch Normalization after Depthwise Convolution.
> 
> * **Why did you use Batch Normalization instead of Layer/Group Norm in your implementation?**
>   ![table 6](https://camo.githubusercontent.com/394ac95b26d06f9942937ab6006fc0a2bb29ec44/68747470733a2f2f696d6775722e636f6d2f6f695841476b582e706e67)
>   In ablation study, you showed LN/GN outperforms BN in ShuffleNetV2 but you didn't discuss or even mention it much.
>   I'm pleased to hear some explanation **why you used BatchNorm instead of other superior methods thorough your experiments(and in your implementation)**.
> 
> Thank you in advance!

Yes we use Batch Normalization by default.

**Comment by omiita** [comment]
@nmaac 
Thank you for quick response!
I see. To be honest, I didn't know other normalization methods are slower than BN. 
This is a super silly question but do you know any reason or resource why other normalization techniques are slow during inference time...? I couldn't find reasons.

Thank you.

**Comment by nmaac** [comment]
> @nmaac
> Thank you for quick response!
> I see. To be honest, I didn't know other normalization methods are slower than BN.
> This is a super silly question but do you know any reason or resource why other normalization techniques are slow during inference time...? I couldn't find reasons.
> 
> Thank you.

You can refer to this [blog](https://towardsdatascience.com/speed-up-inference-with-batch-normalization-folding-8a45a83a89d8) :)

**Comment by omiita** [comment]
Thank you so much!
